src/js/App/Sidenav/__snapshots__/Navigation.test.js.snap

```diff
@@ -244,7 +244,7 @@ exports[`Navigation should render corectly 1`] = `
     ariaRightScroll="Scroll right"
   >
     <NavExpandable
-      className="ins-m-navigation-align"
```

Comment:
This css is required to fix the navigation text left alignment. Kan you keep it? (introduced in https://github.com/RedHatInsights/insights-chrome/pull/958)

Reply:
Sure thing!